app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class='signup-page'>
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="name">
+    <%= f.label :name %>
+    <%= f.text_field :name, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
```

Comment:
- [ ] An error still persists in an attempt to register a new user. Let's ensure to resolve this issue.
NB:
 To be able to analyze this problem correctly, I suggest you clone your version of this project online and check the issue.
 
 **Signup error:**

![image](https://github.com/umair-rao/New-Blog-App/assets/12745474/c62ed052-0db4-45ef-9f17-60724db7244c)

Reply:
I have Clone the repo, installed the node modules and then installed the gem. After that I run the rails server and everything is working perfectly. I have recorded video for your reference, Please check the below link.

https://drive.google.com/file/d/1BxXZGIpIS3VylgU0a3FF5XaU1CIcyX1g/view?usp=sharing